common/reference/baseAddToGlobal.js

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This doesn't scope in JS, in case that's what you wanted to achieve. You have to write it like this:
```javascript
(functiont() {
  var g = ...
  ...
}).call(this);
```
Or you could do it like this:
```javascript
(function(g) {
  g.performance = performance;
  ...
})((typeof global === "object" && global && global["Object"] === Object) ? global : this);
```